src/main/java/sx/blah/discord/handle/obj/IMessage.java

```diff
@@ -189,4 +197,152 @@ public String getUrl() {
 			return url;
 		}
 	}
+
+	/**
+	 * Represents an attachment embedded in the message.
+	 */
+
+	class EmbeddedAttachment {
```

Comment:
Please make this an interface. I do not want beginners to be instantiating this class due to a misunderstanding of the api.